lib/live_debugger/app/debugger/node_state/queries.ex

```diff
@@ -41,6 +45,25 @@ defmodule LiveDebugger.App.Debugger.NodeState.Queries do
     end
   end
 
+  def fetch_node_streams(pid) do
```

Comment:
Private and public functions should usually be groupped together. You can also add spec here